joomla/joomla-cms
# [5.2][Events] Use event classes for Editors and editor buttons plugins

### Summary of Changes

Update `editors` and `editors-xtd` plugins to use `SubscriberInterface` and event classes.


### Testing Instructions

Check editor `None` works as before.
Check the editor buttons, they should work as before.


### Actual result BEFORE applying this Pull Request
Works


### Expected result AFTER applying this Pull Request
Works


### Link to documentations
Please select:
- [x] Pull Request link for manual.joomla.org: https://github.com/joomla/Manual/pull/97

References:
- https://github.com/joomla/joomla-cms/pull/40082
